diff -ur hivex-1.3.10.old/generator/generator.ml hivex-1.3.10/generator/generator.ml
--- hivex-1.3.10.old/generator/generator.ml	2014-03-26 14:56:27.000000000 +0000
+++ hivex-1.3.10/generator/generator.ml	2014-11-17 11:53:46.469958904 +0000
@@ -242,7 +242,7 @@
     "\
 Return the length of the key (name) of a (key, value) pair as produced
 by C<hivex_value_key>. The length can legitimately be 0, so errno is 
-the necesary mechanism to check for errors.
+the necessary mechanism to check for errors.
 
 In the context of Windows Registries, a zero-length name means
 that this value is the default key for this node in the tree.
diff -ur hivex-1.3.10.old/lib/hivex.pod hivex-1.3.10/lib/hivex.pod
--- hivex-1.3.10.old/lib/hivex.pod	2014-01-10 10:00:53.000000000 +0000
+++ hivex-1.3.10/lib/hivex.pod	2014-11-17 11:53:49.288886768 +0000
@@ -346,7 +346,7 @@
 
 Return the length of the key (name) of a (key, value) pair as produced
 by C<hivex_value_key>. The length can legitimately be 0, so errno is 
-the necesary mechanism to check for errors.
+the necessary mechanism to check for errors.
 
 In the context of Windows Registries, a zero-length name means
 that this value is the default key for this node in the tree.
diff -ur hivex-1.3.10.old/perl/lib/Win/Hivex.pm hivex-1.3.10/perl/lib/Win/Hivex.pm
--- hivex-1.3.10.old/perl/lib/Win/Hivex.pm	2014-01-10 10:00:53.000000000 +0000
+++ hivex-1.3.10/perl/lib/Win/Hivex.pm	2014-11-17 11:53:49.337885514 +0000
@@ -212,7 +212,7 @@
 
 Return the length of the key (name) of a (key, value) pair as produced
 by C<value_key>. The length can legitimately be 0, so errno is 
-the necesary mechanism to check for errors.
+the necessary mechanism to check for errors.
 
 In the context of Windows Registries, a zero-length name means
 that this value is the default key for this node in the tree.
diff -ur hivex-1.3.10.old/ruby/ext/hivex/_hivex.c hivex-1.3.10/ruby/ext/hivex/_hivex.c
--- hivex-1.3.10.old/ruby/ext/hivex/_hivex.c	2014-01-13 22:20:34.000000000 +0000
+++ hivex-1.3.10/ruby/ext/hivex/_hivex.c	2014-11-17 11:53:50.808847877 +0000
@@ -580,8 +580,8 @@
  *
  * Return the length of the key (name) of a (key, value)
  * pair as produced by "h.value_key". The length can
- * legitimately be 0, so errno is the necesary mechanism to
- * check for errors.
+ * legitimately be 0, so errno is the necessary mechanism
+ * to check for errors.
  * 
  * In the context of Windows Registries, a zero-length name
  * means that this value is the default key for this node
