From 1c6bd6e0085204425d7c687f2566cb9b13231e6e Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Fri, 17 Feb 2017 15:58:04 +0000
Subject: [PATCH 01/12] Fix Perl directory install path.

---
 perl/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/perl/Makefile.am b/perl/Makefile.am
index a0ae863..144a1bc 100644
--- a/perl/Makefile.am
+++ b/perl/Makefile.am
@@ -43,7 +43,7 @@ $(TESTS): src_deps all
 
 TESTS_ENVIRONMENT = ../run
 
-INSTALLDIRS = site
+INSTALLDIRS = vendor
 
 all: Makefile-pl src_deps
 	$(MAKE) -f Makefile-pl
-- 
1.8.3.1

